setup.cfg: link with pthread.

Signed-off-by: Frederik Aalund <fpa@sbtaqua.com>

--- a/setup.py	2017-04-10 15:38:54.966286285 +0200
+++ b/setup.py	2017-04-10 15:47:50.550286285 +0200
@@ -261,6 +261,8 @@ class uvloop_build_ext(build_ext):
         elif sys.platform.startswith('sunos'):
             self.compiler.add_library('kstat')
 
+        self.compiler.add_library('pthread')
+
         super().build_extensions()
 
 
